data/pathfinder/paizo/roleplaying_game/bestiary_4/b4_races.lst

```diff
@@ -4,7 +4,7 @@ SOURCELONG:Bestiary 4	SOURCESHORT:B4	SOURCEWEB:http://paizo.com/products/btpy91d
-Almiraj								KEY:Almiraj													STARTFEATS:1	SIZE:S	MOVE:Walk,30,Burrow,10				REACH:5																											BONUS:SITUATION|Acrobatics=When Jumping|8|TYPE=Racial											BONUS:SKILL|Perception,Stealth|4|TYPE=Racial												BONUS:STAT|STR|-4	BONUS:STAT|DEX|4	BONUS:STAT|CON|2	BONUS:STAT|INT|-6	BONUS:STAT|CHA|-4					BONUS:VAR|HasLowlightVision|1|TYPE=Base	BONUS:VAR|DarkvisionRange|60|TYPE=Base																																																																																																																										BONUS:WEAPONPROF=Gore|TOHIT,DAMAGE|1		MONCSKILL:Acrobatics|Climb|Fly|Perception|Stealth|Swim																													AUTO:LANG|Sylvan (cannot speak)															ABILITY:Internal|AUTOMATIC|Racial Traits ~ Almiraj							LEGS:4	HANDS:0	NATURALATTACKS:Gore,Natural.Melee.Finesseable.Weapon,*1,2d4																																															MONSTERCLASS:Magical Beast:2			RACETYPE:Magical Beast											TYPE:Magical Beast	CR:1							SOURCEPAGE:p.8							KIT:1|TN					FACT:BaseSize|S
+#Almiraj								KEY:Almiraj													STARTFEATS:1	SIZE:S	MOVE:Walk,30,Burrow,10				REACH:5																											BONUS:SITUATION|Acrobatics=When Jumping|8|TYPE=Racial											BONUS:SKILL|Perception,Stealth|4|TYPE=Racial												BONUS:STAT|STR|-4	BONUS:STAT|DEX|4	BONUS:STAT|CON|2	BONUS:STAT|INT|-6	BONUS:STAT|CHA|-4					BONUS:VAR|HasLowlightVision|1|TYPE=Base	BONUS:VAR|DarkvisionRange|60|TYPE=Base																																																																																																																										BONUS:WEAPONPROF=Gore|TOHIT,DAMAGE|1		MONCSKILL:Acrobatics|Climb|Fly|Perception|Stealth|Swim																													AUTO:LANG|Sylvan (cannot speak)															ABILITY:Internal|AUTOMATIC|Racial Traits ~ Almiraj							LEGS:4	HANDS:0	NATURALATTACKS:Gore,Natural.Melee.Finesseable.Weapon,*1,2d4																																															MONSTERCLASS:Magical Beast:2			RACETYPE:Magical Beast											TYPE:Magical Beast	CR:1							SOURCEPAGE:p.8							KIT:1|TN					FACT:BaseSize|S
```

Comment:
Why block out? - Should be removed if not needed.

Reply:
It's one of the races for improved familiars. Should probably delete the stats and keep the name with a note of what file it's in.